feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -75,6 +78,11 @@ fun PreviewScreen(
     viewModel: PreviewViewModel = hiltViewModel<PreviewViewModel, PreviewViewModel.Factory>
         { factory -> factory.create(previewMode) }
 ) {
+    // start trace between preview screen starting and the earliest possible completed capture
+    // todo: only run traces on a specific build version or flavor(?)
+    if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.Q) {
+        Trace.beginAsyncSection(FIRST_FRAME_TRACE, FIRST_FRAME_COOKIE)
```

Comment:
If you want a separate trace for the PreviewScreen -> First Frame, you'll need to use a separate cookie if you're using the same trace tag.

If you're able to plumb up a callback as I mentioned in another comment, it will make this cookie management easier since you can have 2 separate cookies for `endAsyncSection`.

Reply:
So I didn't want to separate the two traces. The issue i kept finding was that the first frame tests using `StartupMode.HOT` weren't being found. The trace start wasnt being called, so I needed to be able to call it in the second best location, which I think is here.

Calling beginAsync on top of each other using the same trace and cookie should be ignored, so there aren't issues with cold startups where it would be "called" twice.